### ISSUE TITLE
in_channels -> out_channels

### ISSUE THREAD

**Issue by bub3690** [YoojLee/paper_review#54]
https://github.com/YoojLee/paper_review/blob/b1988364aac5c8404b1fb708edf47f5c1f8106eb/code/cycle_gan/model.py#L30C23-L30C27

in_channels -> out_channels ?
`
nn.Conv2d(self.in_channels,self.out_channels,kernel_size=3, padding='same', padding_mode='reflect')
`

**Comment by YoojLee** [comment]
sorry what is your question?

**Comment by bub3690** [comment]
Thank you for reply.

In class "ResidualBlock".
`
self.block = nn.Sequential(
            nn.Conv2d(self.in_channels, self.out_channels, kernel_size=3, padding='same', padding_mode='reflect'),
            nn.InstanceNorm2d(self.out_channels),
            nn.ReLU(inplace=True),
            nn.Conv2d(self.in_channels, self.out_channels, kernel_size=3, padding='same', padding_mode='reflect'),
            nn.InstanceNorm2d(self.out_channels)
        )
`
I think there is error in Conv2d parameters.

Because, bottle neck should be 'in_channel -> out_channel' & 'out_channel -> out_channel ' .

So it should be this.
`
self.block = nn.Sequential(
            nn.Conv2d(self.in_channels, self.out_channels, kernel_size=3, padding='same', padding_mode='reflect'),
            nn.InstanceNorm2d(self.out_channels),
            nn.ReLU(inplace=True),
            nn.Conv2d(**self.out_channel** , self.out_channels, kernel_size=3, padding='same', padding_mode='reflect'),
            nn.InstanceNorm2d(self.out_channels)
        )
`

**Comment by YoojLee** [comment]
Thanks for letting me know! Have you observed any improvement in performance then? :) Thanks!
